csharp/src/Apache.Arrow.Adbc/C/CAdbcDriverExporter.cs

```diff
@@ -708,13 +754,8 @@ public void Dispose()
                 connection = null;
             }
 
-            public unsafe AdbcStatusCode GetObjects(ref CAdbcConnection nativeConnection, int depth, byte* catalog, byte* db_schema, byte* table_name, byte** table_type, byte* column_name, CArrowArrayStream* cstream, ref CAdbcError error)
+            public unsafe void GetObjects(ref CAdbcConnection nativeConnection, int depth, byte* catalog, byte* db_schema, byte* table_name, byte** table_type, byte* column_name, CArrowArrayStream* cstream)
```

Comment:
why the switch to void? I believe this has to return the AdbcStatusCode per https://github.com/apache/arrow-adbc/blob/3f80831d12b6e5a78a4321f67e28d652951241cf/adbc.h#L1609.